app/Jobs/ExportAccountAsJson.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Helpers\DBHelper;
+use App\Models\User\User;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Foundation\Bus\Dispatchable;
+
+class ExportAccountAsJson
+{
+    use Dispatchable, SerializesModels;
+
+    protected $file = '';
+    protected $path = '';
+
+    /**
+     * Storage disk used to store the exported file.
+     * @var string
+     */
+    public const STORAGE = 'public';
+
+    /**
+     * Create a new job instance.
+     *
+     * @param string|null $file
+     * @param string|null $path
+     */
+    public function __construct($file = null, $path = null)
+    {
+        $this->path = $path ?? 'exports/';
+        $this->file = $file ?? rand().'.json';
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return string
+     */
+    public function handle()
+    {
+        $downloadPath = $this->path.$this->file;
+
+        /** @var User $user */
+        $user = auth()->user();
+        $account = $user->account;
+
+        $json_output = '';
+        $this->generate_json_header($json_output, $user);
```

Comment:
Could you use the camelCase for your method names? We try to use it everywhere in the repository. This apply to all methods in this PR.

Reply:
Thank you for the review! I have updated the function names.